### PR TITLE
Editor page is reporting Errors now, which can be fixed later

### PR DESCRIPTION
**Pull request by tim-boerner**

### CODE REVIEW

**Comment by henning-gerhardt** [comment]
Forget to mentioned that the XHTML files

- Metadaten2oben.xhtml
- Metadaten3links.xhtml
- Metadaten2rechts.xhtml

must be adjusted too as they are now included into the main file like other included files.

**Comment by tim-boerner** [comment]
Doctype has been adjusted

**Comment by solth** [comment]
The changes in https://github.com/kitodo/kitodo-production/pull/941 are required in order to test and review this pull request.

**Comment by henning-gerhardt** [comment]
@solth: as pull request #941 is merged now, can you review this pull request? With this changes I get at least errors displayed instead of empty frame set windows.

**Comment by solth** [comment]
If you encounter an error like the following:
```
javax.el.ELException - The identifier [class] is not a valid Java identifier as required by section 1.19 of the EL specification 
```
it can be fixed by replacing `member.class.simpleName` with `member['class'].simpleName`.

**Comment by henning-gerhardt** [comment]
@tim-boerner : can you change it like @solth proposing as I get this kind of error too.

**Comment by tim-boerner** [comment]
Replaced `member.class.simpleName` with `member['class'].simpleName` as @solth proposed.
Changed `Metadaten.paginierungArt` to `Metadaten.paginationType` as it was already changed in Metadaten-Class.
Inserted some TODOs for replacing of old jsf library elements (jenia4faces).

Current Error is now:

> java.lang.NullPointerException - Error reading [newMetadataGroup] on type [de.sub.goobi.metadaten.Metadaten]
> at de.sub.goobi.metadaten.Metadaten.getNewMetadataGroup(Metadaten.java:3269)
> 
> /pages/incMeta/addMetadataGroup.xhtml at line 46 and column 41 value="#{Metadaten.newMetadataGroup.members}"

File: Kitodo/src/main/webapp/pages/Metadaten2rechts.xhtml [hunk 1]
```diff
@@ -26,84 +25,86 @@
 
         <h:body style="margin: 0px 2px 2px 2px;" class="metadatenRechtsBody" onload="addableTypenAnzeigen();TreeReloaden()">
 
+            <!--TODO: Check this Script, it results in empty page with no error-->
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/Metadaten2rechts.xhtml]: Please move this TODO line two lines down to the out commented part and adjust indent.

File: Kitodo/src/main/webapp/pages/incMeta/Strukturdaten.xhtml [hunk 2]
```diff
@@ -147,87 +147,90 @@
...
-                                        <h:commandLink target="links" immediate="false"
-                                                       action="#{Metadaten.addAdditionalOpacPpns}"
-                                                       title="#{msgs.uebernehmen}"
-                                                       onclick="if (!confirm('#{msgs.wirklichAusfuehren}?')) return">
-                                            <h:outputText value="#{msgs.alsUnterelemente}"/>
-                                        </h:commandLink>
-                                    </h:panelGroup>
-                                </h:panelGrid>
-                            </jd:hideableArea>
+                                    <!--<h:selectOneMenu id="katalogauswahl" value="#{Metadaten.opacCatalog}"-->
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/pages/incMeta/Strukturdaten.xhtml]: Please add an TODO for fixing this in future.

**Pull request merged**